README.md

```diff
@@ -337,23 +336,26 @@ Also for local development:
 - GOOGLE_APPLICATION_CREDENTIALS must be set to a file that is mounted, containing GSA credentials.
 - Alternatively, a KUBECONFIG file must be set and configured for the intended cluster.
 
-# How it works
+## How it works
 
-The setup is similar with Istio on VM support.
+The setup is similar to Istio on VM support.
 
-The 'mesh connector' is an extended version of the "East-West" Gateway, provide access to the in-cluster Istiod and
+The `mesh connector` is an extended version of the "East-West" Gateway, provide access to the in-cluster Istiod and
 handles forwarding the requests from pods to CloudRun.
 
 The 'golden image' includes the istio sidecar components and a launcher, similar with Istio on VMs. The launcher will
-load the istio-system/mesh-env config map, created by the connector (until it is moved to Istiod proper), set iptables
-interception, generate the mesh certificates and start envoy - using pilot-agent, and finally run the application.
+1. Load the istio-system/mesh-env config map created by the connector (until it is moved to Istiod proper)
+1. Set iptables interception
+1. Generate the mesh certificates
+1. Start envoy using pilot-agent
+1. Run the application.
 
-The application works the same as pods or VMs with Istio support - outgoing requests are intercepted, sent to Envoy
+The application works the same as Pods or VMs with Istio support: outgoing requests are intercepted then sent to Envoy
 where mTLS and policies are handled.
 
-For requests from Pods to CloudRun, the WorkloadGroup and other configs created by
+For requests from Pods to CloudRun, the WorkloadGroup and other configs created by [TODO this sentence is incomplete]
```

Comment:
Will remove it for now - and add it back when auto-registration is implemented...